tests/models/tvp/test_processor_tvp.py

```diff
@@ -0,0 +1,81 @@
+import inspect
+import tempfile
+import unittest
+
+import numpy as np
+
+from transformers import TvpProcessor
+from transformers.testing_utils import require_torch, require_vision
+
+from ...test_processing_common import ProcessorTesterMixin
+
+
+class TvpProcessorTest(ProcessorTesterMixin, unittest.TestCase):
+    from_pretrained_id = "Jiqing/tiny-random-tvp"
+    processor_class = TvpProcessor
+    videos_data_arg_name = "pixel_values"
+
+    def setUp(self):
+        self.tmpdirname = tempfile.mkdtemp()
+        processor = self.processor_class.from_pretrained(self.from_pretrained_id)
+        processor.save_pretrained(self.tmpdirname)
+
+    @require_vision
+    def prepare_video_inputs(self):
+        return [np.random.randint(255, size=(3, 30, 400), dtype=np.uint8)]
+
+    @require_torch
+    @require_vision
+    def test_video_processor_defaults_preserved_by_kwargs(self):
+        if "video_processor" not in self.processor_class.attributes and (
+            "videos" not in inspect.signature(self.processor_class.__call__).parameters
+            or inspect.signature(self.processor_class.__call__).parameters["videos"].annotation == inspect._empty
```

Comment:
Is this because of InstructBlipVideo? If it's only that, we better override this test for InstructBlipVideo and leave the check as `"video_processor" in classes`

When overriding for TVP, there's no need to check as we know if model has video processor or not. Actually seems like it's gonna be skipped because TVP has only `image_processor`

Reply:
This is for:
- InstructBlipVideo
- TVP
- Video Llava, &
- X-Clip

Only Llava-Next-Video actually has a `video_processor_class`

Reply:
I'm leaning on leaving this here cuz it covers more models than the original check + it significantly reduces LoCs